src/test/java/com/xebialabs/overthere/cifs/CifsFileTest.java

```diff
@@ -65,15 +64,15 @@ public void setupOptions() {
     @Test
     public void shouldReturnNullForParentFileOfRoot() {
         options.set(USERNAME, "user@domain.com");
-        CifsWinRmConnection cifsWinRmConnection = new CifsWinRmConnection(CIFS_PROTOCOL, options, INSTANCE);
+        CifsProcessConnection cifsWinRmConnection = new CifsProcessConnection(CIFS_PROTOCOL, options, INSTANCE);
```

Comment:
Hmm totally forgot, this refactoring breaks backwards compatibility. Maybe keep using the old classes for Cifs for this release and return them from the CifsConnectionBuilder.